master/getting-started/openstack/installation/redhat.md

```diff
@@ -73,40 +89,16 @@ cluster, please get in touch with us and we'll be happy to help you
 through the process.
 
 1.  Install and configure etcd.
```

Comment:
I think we should say that etcd v3 is required here.  It seems very common for users to be manually controlling package versions in the RH world.

Reply:
I suggest presenting these instructions as a single-node quick-start, but referring the user to etcd's docs, for how to set up a production cluster if needed.